internal/service/accesslistapikey/resource_access_list_api_key_migration_test.go

```diff
@@ -57,9 +57,54 @@ func TestAccMigrationProjectAccesslistAPIKey_SettingCIDRBlock(t *testing.T) {
 		PreCheck:     func() { mig.PreCheckBasic(t) },
 		CheckDestroy: checkDestroy,
 		Steps: []resource.TestStep{
+			{
+				ExternalProviders: acc.ExternalProviders("1.14.0"), // testing provider version before this resource was migrated to Atlas SDK
+				Config:            configWithCIDRBlock(orgID, description, cidrBlock),
+				Check: resource.ComposeTestCheckFunc(
+					checkExists(resourceName),
+					resource.TestCheckResourceAttr(resourceName, "org_id", orgID),
+					resource.TestCheckResourceAttr(resourceName, "cidr_block", cidrBlock),
+				),
+			},
 			{
 				ExternalProviders: mig.ExternalProviders(),
 				Config:            configWithCIDRBlock(orgID, description, cidrBlock),
+				ConfigPlanChecks: resource.ConfigPlanChecks{
+					PreApply: []plancheck.PlanCheck{
+						acc.DebugPlan(),
+						plancheck.ExpectEmptyPlan(),
+					},
+				},
+			},
+			{
+				ProtoV6ProviderFactories: acc.TestAccProviderV6Factories,
+				Config:                   configWithCIDRBlock(orgID, description, cidrBlock),
+				ConfigPlanChecks: resource.ConfigPlanChecks{
+					PreApply: []plancheck.PlanCheck{
+						acc.DebugPlan(),
+						plancheck.ExpectEmptyPlan(),
+					},
+				},
+			},
+		},
+	})
+}
+
+func TestAccMigrationProjectAccesslistAPIKey_SettingCIDRBlock_WideCIDR_SDKMigration(t *testing.T) {
+	var (
+		resourceName = "mongodbatlas_access_list_api_key.test"
+		orgID        = os.Getenv("MONGODB_ATLAS_ORG_ID")
+		cidrBlock    = "100.10.0.0/16"
+		description  = acc.RandomName()
+	)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { mig.PreCheckBasic(t) },
+		CheckDestroy: checkDestroy,
+		Steps: []resource.TestStep{
+			{
+				ExternalProviders: acc.ExternalProviders("1.14.0"), // testing provider version before this resource was migrated to Atlas SDK
```

Comment:
consider using SkipIfVersionBelow and/or IsProviderVersionAtLeast instead.
the issue with  acc.ExternalProviders is that it will always check against that version ignoring MONGODB_ATLAS_LAST_VERSION, that can be confusing.

Reply:
While this is true, I think we have to revise the provider versions we are passing for running migrations tests in the CI (https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/.github/workflows/code-health.yml#L91) and test suite. As of the moment we are simply running against latest so any test using `SkipIfVersionBelow` is likely being skipped if it is not the latest release.

Reply:
While this is true, I think we have to revise the provider versions we are passing for running migrations tests in the CI (https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/.github/workflows/code-health.yml#L91) and test suite. As of the moment we are simply running against latest so any test using SkipIfVersionBelow is likely being skipped if it is not the latest release.

Reply:
using SkipIfVersionBelow